src/data/fonts.ts

```diff
@@ -4,5 +4,5 @@
 export default {
   "garamond-regular": "AGaramondPro-Regular",
   "garamond-italic": "AGaramondPro-Italic",
-  "avant-garde-regular": "Avant Garde Gothic ITCW01Dm",
+  "avant-garde-regular": "AvantGardeGothicITC",
```

Comment:
Will this work with the OSS fonts too? Or should this file return a different name for OSS users?

Reply:
It should be fine because I manually changed the metadata of the OSS avant garde to match it.

Reply:
Ah yes, I see 👍

Reply:
I'll try it out in OSS mode just to be sure though.